fuse.py

```diff
@@ -412,6 +455,20 @@ class fuse_file_info(ctypes.Structure):
             ('padding', ctypes.c_uint, 29),
             ('fh', ctypes.c_uint64),
             ('lock_owner', ctypes.c_uint64)]
+elif _system == "OpenBSD":
+    class fuse_file_info(ctypes.Structure):
+        _fields_ = [
+            ('flags', ctypes.c_int32),
+            ('fh_old', ctypes.c_uint32),
+            ('writepage', ctypes.c_int32),
+            ('direct_io', ctypes.c_uint32, 1),
+            ('keep_cache', ctypes.c_uint32, 1),
+            ('flush', ctypes.c_uint32, 1),
+            ('nonseekable', ctypes.c_uint32, 1),
+            ('padding', ctypes.c_uint32, 27),
+            ('flock_release', ctypes.c_uint32, 1),
```

Comment:
Why is this reordered after `padding`? For all other systems, this bit is before the padding. I doubt that this is something OpenBSD-specific.